modules/tinymce/src/core/main/ts/keyboard/DeleteBackspaceKeys.ts

```diff
@@ -50,24 +51,54 @@ const executeKeydownOverride = (editor: Editor, caret: Cell<Text | null>, evt: K
   });
 };
 
-const executeKeyupOverride = (editor: Editor, evt: KeyboardEvent) => {
+const executeKeyupOverride = (editor: Editor, evt: KeyboardEvent, isBackspaceKeydown: boolean) => {
+  const platform = PlatformDetection.detect();
+  const os = platform.os;
+  const browser = platform.browser;
+  const multiDeleteKeyPatterns: MatchKeys.KeyPattern[] = os.isMacOS() ? [
+    { keyCode: VK.BACKSPACE, altKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) },
+    { keyCode: VK.DELETE, altKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) }
+  ] : [
+    { keyCode: VK.BACKSPACE, ctrlKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) },
+    { keyCode: VK.DELETE, ctrlKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) }
+  ];
+
+  // macOS surpresses keyup events for most keys including Backspace when Meta key is engaged
+  // To emulate Meta + Backspace on macOS, add a pattern for the meta key when backspace was
+  // detected on keydown
```

Comment:
Do we have some research to back this up? I checked in Slack and nobody responded to your question, sorry about that.

(I also wrote this comment in December and forgot to submit my review, oops)

Reply:
I think Cmd + Backspace, like most other Cmd-modified keys, is transformed into a "key equivalent" on `keydown`, and so Cmd + Backspace does not fire a `keyup`. See "Handling Key Equivalents" in https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/EventOverview/HandlingKeyEvents/HandlingKeyEvents.html

Reply:
@TheSpyder I think Cmd + Backspace, like most other Cmd-modified keys, is transformed into a "key equivalent" on `keydown`, and so Cmd + Backspace does not fire a `keyup`. See "Handling Key Equivalents" in https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/EventOverview/HandlingKeyEvents/HandlingKeyEvents.html

Reply:
well TIL, I guess. I didn't think keydown without keyup was possible, but I've confirmed in an [event tester](https://javascript.info/keyboard-events)

```
keydown key=Meta code=MetaLeft metaKey
--------------------------------------------------------------------------------
keydown key=Backspace code=Backspace metaKey
--------------------------------------------------------------------------------
keyup key=Meta code=MetaLeft
```